docs/components/arm/_index.md

```diff
@@ -66,18 +66,15 @@ The [arm API](/appendix/apis/components/arm/) supports the following methods:
 
 {{< readfile "/static/include/components/apis/generated/arm-table.md" >}}
 
-## Motion planning with your arm's software
+## Motion planning with your arm
 
-Each arm model implements the [Arm API](#api).
+Each arm model provided by Viam wraps the arm manufacturer's API with the Viam [arm API](#api).
```

Comment:
This isn't necessarily true. Some arms directly interact with servos, or something else.